lib/Controller/RoomController.php

```diff
@@ -817,14 +817,19 @@ public function setNotificationCalls(int $level): DataResponse {
 	 * Rename a room
 	 *
 	 * @param string $roomName New name
-	 * @return DataResponse<Http::STATUS_OK, TalkRoom, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, array{error: 'type'|'value'}, array{}>
+	 * @return DataResponse<Http::STATUS_OK, TalkRoom, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, array{error: 'type'|'value'}, array{}>|DataResponse<Http::STATUS_FORBIDDEN, array{error: 'event'}, array{}>
 	 *
 	 * 200: Room renamed successfully
 	 * 400: Renaming room is not possible
+	 * 403: Renaming room is not possible - it can only be edited by the associated calendar event
```

Comment:
I'd prefer to stick with `400 Bad Request` for all errors
It's no big difference to 400 with error 'type'